RadixWallet/Features/NewConnectionFeature/Coordinator/NewConnection+View.swift

```diff
@@ -76,10 +76,11 @@ extension View {
 			Text(title)
 				.foregroundColor(.app.gray1)
 				.textStyle(.sheetTitle)
-
-			Spacer(minLength: 0)
+				.padding(.bottom, .medium3)
```

Comment:
The `VStack` that holds this doesn't set any spacing, I would set it to `.zero` so we make sure the distance between title and content is `.medium3`